src/components/BlueButton.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <header class="bg-white">
+    <div class="max-w-7xl mx-auto py-2 sm:px-6 lg:px-8 text-black text-left">
+      <h1 :class="getClass()">
```

Comment:
Why is the button wrapped in a header tag, and why does it contain an h1 element? Generally you should only expect one h1 element per page.

Although I can't find out where this actually used, so maybe I'm misunderstanding something. (The name and content cerainly makes it look more like a heading.

Is it just completely misnamed?

Reply:
Sorry, that BlueButton class is not being used. It was started off with an idea to have a button, but later changed. That shouldn't have gone to commit.